SyncTelemetryTests/FxALoginRegistrationTelemetryTests.swift

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+import Shared
+@testable import Client
+
+struct MockFxAUrls {
+    static let mockSigInUrl = URL(string: "https://accounts.firefox.com/oauth/signin?action=email")!
+    static let mockSignUpUrl = URL(string: "https://accounts.firefox.com/oauth/signup?action=email")!
+    static let mockConfirmSignUpCodeUrl = URL(string: "https://accounts.firefox.com/oauth/confirm_signup_code")!
+    // Added / in the end as some urls end with a slash
+    static let mockSignInTockenUrl =  URL(string: "https://accounts.firefox.com/oauth/signin_token_code/")!
+}
+
+class SyncTelemetryTests: XCTestCase {
+
+    var fxaWebViewTelemetry: FxAWebViewTelemetry!
+
+    override func setUp() {
+        super.setUp()
+        fxaWebViewTelemetry = FxAWebViewTelemetry()
+    }
+
+    override func tearDown() {
+        fxaWebViewTelemetry = nil
+        super.tearDown()
+    }
+
+    func testSignInFlow() {
+        let flow = fxaWebViewTelemetry.getFlowFromUrl(fxaUrl: MockFxAUrls.mockSigInUrl)
+        XCTAssertNotNil(flow)
+        XCTAssert(flow == FxAUrlPathStartedFlow.signinStarted)
```

Comment:
We can use `XCTAssertEqual` to assert for equality, make it a bit clearer on Bitrise when/if the test fails

Reply:
Oh good call, let me change that before merging.